pynest/nest/lib/hl_api_helper.py

```diff
@@ -69,7 +69,8 @@
 # deprecated functions will be registered by the @deprecated decorator.
 _deprecation_warning = {'BackwardCompatibilityConnect': True,
                         'subnet': True,
-                        'aeif_cond_alpha_RK5': True}
+                        'aeif_cond_alpha_RK5': True,
+                        'iaf_psc_alpha_canon': True}
```

Comment:
Also deprecated `iaf_psc_delta_canon`, it should be replaced with `iaf_psc_delta_ps` eventually.